hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/FlushPolicy.java

```diff
@@ -17,7 +17,10 @@
  */
 package org.apache.hadoop.hbase.regionserver;
 
+import java.util.ArrayList;
```

Comment:
Should remove these imports? Seems we add a method in this class and then moved it out but forgot to remove these imports?

Reply:
Oh, you are right, fixed.
Thanks.